go.mod

```diff
@@ -44,6 +44,7 @@ require (
 	github.com/xeipuuv/gojsonschema v1.2.0
 	golang.org/x/crypto v0.0.0-20200622213623-75b288015ac9
 	golang.org/x/mod v0.3.0
+	golang.org/x/net v0.0.0-20210813160813-60bc85c4be6d
```

Comment:
is this really used? or dead now in the final commit?

Reply:
This is indeed needed: https://github.com/anchore/syft/blob/c90ab50402d13ec69609fa80b6d11622689fdb30/syft/pkg/cataloger/java/parse_pom_xml.go#L19-L21